src/app/core/services/authorization.service.ts

```diff
@@ -1,28 +1,38 @@
 import { Injectable } from '@angular/core';
 import { Router } from '@angular/router';
-import { LoginForm } from '../models/authorization';
-import { User } from '../models/user';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 
-const fakeUser: User = {
-  id: 1,
-  firstName: 'FirstName',
-  lastName: 'LastName',
-  email: '',
-};
+import { LoginForm, Token } from '../models/authorization';
+import { User } from '../models/user';
+import { catchError, Observable, tap, throwError } from 'rxjs';
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthorizationService {
+  private apiUrl = 'http://localhost:3004/auth';
   public isAuth: boolean = !!localStorage.getItem('accessToken');
```

Comment:
isAuth varibale is not safe here... anyone can modify it outside the service, make it private.